### ISSUE TITLE
Issue with postinstall removing related dependencies

### ISSUE THREAD

**Issue by alexvicegrab** [porsager/env-install#1]
When I do postinstall, the npm install command to use a GITHUB_API_TOKEN to install a private repository, the subsequent npm install moves some of the dependencies such that I they cannot be found anymore.

**Comment by porsager** [comment]
Sorry for the late response.

Which dependencies are moved, and where are they moved, or are they removed? Would you be able to make a reproducible setup?

**Comment by alexvicegrab** [comment]
I can't make a reproducible setup because the repositories are private. But what I can say is that the post-install repository had some of the same dependencies as another public repository. The post-install appeared to move some of the dependencies from one repository in the root node_modules to the other.

I ended up setting up a custom Docker image where I inserted the relevant GITHUB_API_TOKEN values to the private repo, installing, replacing the original package*.json files and squashing the image to prevent private info leaking.

But thanks for the response.
